tests/Microsoft.Identity.Web.Test/TestTelemetryClient.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Abstractions;
+
+namespace Microsoft.Identity.Web.Test
+{
+    internal class TestTelemetryClient : ITelemetryClient
```

Comment:
We want to deprecate `ITelemetryClient`. Can you please re-write the test to not use this?

Reply:
I cant seems to find an easy way to get the cache telemetry data without some kind of telemetry client. Maybe it is better to update the tests when the new telemetry client is released. Atleast this is able to validate its functionality for now.

Reply:
CC @jennyf19  @jmprieur